src/screens/FeedScreen/ContentPoll.js

```diff
@@ -287,7 +288,7 @@ const stylesComponent = (pollLength) =>
       height: 12,
       alignSelf: 'center',
       borderRadius: 6,
-      borderColor: COLORS.black,
+      borderColor: colors.black,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color property has been updated from `COLORS.black` to `colors.black`. Make sure that `colors.black` is defined and is the intended color.
```diff
- borderColor: COLORS.black
+ borderColor: colors.black
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->